apps/sched/sched.js

```diff
@@ -1,10 +1,81 @@
+/* eslint-disable no-unused-vars */
+
 // Chances are boot0.js got run already and scheduled *another*
 // 'load(sched.js)' - so let's remove it first!
 if (Bangle.SCHED) {
   clearInterval(Bangle.SCHED);
   delete Bangle.SCHED;
 }
 
+function formatMS(ms) {
+  if (ms < 60000) {
+    // less than a minute → show seconds
+    return Math.round(ms / 1000) + "s";
+  } else {
+    // one minute or more → show minutes
+    return Math.round(ms / 60000) + "m";
+  }
+}
+
+function showSnoozeMenu(alarm){
+  const alarmIndex = alarms.indexOf(alarm);
+  const settings = require("sched").getSettings();
+  let buzzCount = 0;
+  let buttons=
+  Bangle.buzz(40);
+  if(alarm.timer){
+    
+    let timerLength=alarm.timer
+    let buttons={ "15s": 15, "30s":30,"1m":60 ,"2m":120,"5m":360};
+    let formattedLength = formatMS(timerLength)+"*";
+    buttons[formattedLength] = Math.round(timerLength/1000);
+    //different button lengths
+    E.showPrompt("Choose snooze length", {
+      title: "Snooze Options",
+      buttons: buttons
```

Comment:
I don't mind about this, just letting you know JS allows a little shorthand:

```suggestion
      buttons
```